src/course-outline/data/slice.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const slice = createSlice({
+  name: 'courseOutline',
+  initialState: {
+    reindexLink: '',
```

Comment:
I think we need to create `courseOutlineData` key to store whole response. Because when you need a new key from a response, you'll have to create a new one in both files, here and `./slice.js` too.

Reply:
no need to add a new key